Source/ConstraintMakerPriortizable.swift

```diff
@@ -30,12 +30,68 @@
 
 public class ConstraintMakerPriortizable: ConstraintMakerFinalizable {
     
+    public enum Priority: Strideable, RawRepresentable, Equatable {
+        case low
+        case medium
+        case high
+        case required
+        case custom(value: Float)
+        
+        public init?(rawValue: Float) {
+            if rawValue < 0 || rawValue > 1000 {
+                return nil
+            }
+            
+            self = .custom(value: rawValue)
+        }
+        
+        var macOSMedium: Float {
```

Comment:
I would make it private or use it inline directly. The naming could be something different ? or is this because other platform don't really have medium priority? (AFAIK)